### DISCUSSION TITLE
Readme wrong lazy config example

### DISCUSSION THREAD

**Discussion by agusdmb** [olimorris/persisted.nvim#69] [General]
Hey, great plugin here! thanks for all the work

I saw this minor issue while going through the readme.

The current example doesn't work on lazy
```lua
-- Lua
use({
  "olimorris/persisted.nvim"
  config = true
})
```

should be:

```lua
-- Lua
{
  "olimorris/persisted.nvim",
  config = true
}
```

i wanted to add an issue for this but it seems like a lot of effort for something this small (creating a whole PR too)

**Comment by olimorris** [comment]
Thank you!

**Comment by agusdmb** [comment]
@olimorris hey there!, the fix only added the comma. it didn't remove the `use` and the parenthesis
